rest/model/task.go

```diff
@@ -77,6 +78,7 @@ func (at *APITask) BuildFromService(t interface{}) error {
 	case *task.Task:
 		(*at) = APITask{
 			Id:            APIString(v.Id),
+			ProjectId:     APIString(v.Project),
```

Comment:
You'll need to update ToService as well.

Reply:
What should I update here? In the opposite direction, `APITask` gets both `ProjectId` and `Branch` from `Task.Project`